src/main.ts

```diff
@@ -22,6 +22,7 @@ export async function _main(): Promise<void> {
 			GatewayIntentBits.DirectMessages,
 			GatewayIntentBits.GuildMessageTyping,
 			GatewayIntentBits.GuildVoiceStates,
+			GatewayIntentBits.MessageContent,
```

Comment:
This requires an update in the developer portal to allow this Intent, but without it, the bot can't read messages to make a starboard post from them.